### DISCUSSION TITLE
Regarding data storage

### DISCUSSION THREAD

**Discussion by codysaint** [FlowiseAI/Flowise#1762] [Q&A] [answered]
Hi, I just started with flowise and it seems amazing.

I wanted to know where **my data (pdf/txt/etc.)/text embeddings** are stored when I upload it via **Flowise** canvas on **local**. I have used **docker-compose** to run the platform.

![image](https://github.com/FlowiseAI/Flowise/assets/113966422/460c7a26-1c7e-4f1b-a3e2-0785042e5d6f)

**Comment by AutomationRoyalty** [comment]
I asked ChatGPT to answer your question and it did. Would you like me to send you the full response as I don't want to be spamming it here?

**Reply by lancerkind** [reply, marked as answer]
Interesting. What GPT are you using? The 3.5 version doesn't know Flowise.

**Comment by lancerkind** [comment]
CodySaint, it depends on what kind of storage/database node you added to your flow. If you used an in-memory DB, then it's stored locally on your computer. If you used a remote database, it will be stored there. So you need to look at your flow.

**Reply by codysaint** [reply]
oh got it. Thanks for that clarity @lancerkind :-) 👍

**Comment by codysaint** [comment]
Also, I wanted to use local models in flowise and as I went to the document I came across LocalAI but when I am running it using docker-compose i am getting below error:

LocalAI version: latest

Docker desktop: 4.27.2

Do you have any idea about this @lancerkind ?


![image](https://github.com/FlowiseAI/Flowise/assets/113966422/af8e0d41-3757-4257-b029-8074543bb46a)
